src/internal/operators/shareReplay.ts

```diff
@@ -5,6 +5,42 @@ import { MonoTypeOperatorFunction, SchedulerLike } from '../types';
 import { Subscriber } from '../Subscriber';
 
 /**
+ * Share source and replay specified number of emissions on subscription.
+ *
+ * This operator is a specialization of `replay` that connects to the connectable observable sequence
```

Comment:
`... that connects to a source observable and multicasts through a \`ReplaySubject\` constructed with the specified arguments.`

Reply:
Done.